src/main/java/mate/academy/service/CinemaHallService.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.service;
+
+import java.util.List;
+import mate.academy.lib.Service;
+import mate.academy.model.CinemaHall;
+
+@Service
```

Comment:
Why you mark interface and realization by anno @Service?

Reply:
miss click